Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -3595,7 +3604,787 @@ script:
+      description: The source of the scan initiation.
+      type: String
+    - contextPath: CrowdStrike.ODSScan.quarantine
+      description: If was set to quarantine.
+      type: Boolean
+    - contextPath: CrowdStrike.ODSScan.cpu_priority
+      description: The CPU priority for the scan (1-5).
+      type: Number
+    - contextPath: CrowdStrike.ODSScan.preemption_priority
+      description: The preemption priority for the scan.
+      type: Number
+    - contextPath: CrowdStrike.ODSScan.metadata.host_id
+      description: A unique identifier for the host that was scanned.
+      type: String
+    - contextPath: CrowdStrike.ODSScan.metadata.host_scan_id
+      description: A unique identifier for the scan that was performed on the host.
+      type: String
+    - contextPath: CrowdStrike.ODSScan.metadata.scan_host_metadata_id
+      description: A unique identifier for the metadata associated with the host scan.
+      type: String
+    - contextPath: CrowdStrike.ODSScan.metadata.filecount.scanned
+      description: The number of files that were scanned.
+      type: Number
+    - contextPath: CrowdStrike.ODSScan.metadata.filecount.malicious
+      description: The number of files that were identified as malicious.
+      type: Number
+    - contextPath: CrowdStrike.ODSScan.metadata.filecount.quarantined
+      description: The number of files that were quarantined.
+      type: Number
+    - contextPath: CrowdStrike.ODSScan.metadata.filecount.skipped
+      description: The number of files that were skipped during the scan.
+      type: Number
+    - contextPath: CrowdStrike.ODSScan.metadata.filecount.traversed
+      description: The number of files that were traversed during the scan.
+      type: Number
+    - contextPath: CrowdStrike.ODSScan.metadata.status
+      description: The status of the scan on this host. (e.q., "pending", "running", "completed", or "failed").
+      type: String
+    - contextPath: CrowdStrike.ODSScan.metadata.started_on
+      description: The date and time that the scan started.
+      type: Date
+    - contextPath: CrowdStrike.ODSScan.metadata.completed_on
+      description: The date and time that the scan completed.
+      type: Date
+    - contextPath: CrowdStrike.ODSScan.metadata.last_updated
+      description: The date and time that the metadata was last updated.
+      type: Date
+    - contextPath: CrowdStrike.ODSScan.status
+      description: The status of the scan. (e.q., "pending", "running", "completed", or "failed").
+      type: String
+    - contextPath: CrowdStrike.ODSScan.hosts
+      description: A list of the host IDs that were scanned.
+      type: String
+    - contextPath: CrowdStrike.ODSScan.endpoint_notification
+      description: A boolean value indicating whether endpoint notifications are enabled.
+      type: Boolean
+    - contextPath: CrowdStrike.ODSScan.pause_duration
+      description: The number of minutes to pause between scanning each file in hours.
```

Comment:
Not sure about this description.
Its the "number of minutes", but what does it mean "in hours"?